test/ida_rsa

```diff
@@ -0,0 +1,38 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Once in a while this will fire in various security scanners and reports. I would rather generate the key on the fly in a test to avoid the noise

Reply:
That would be ideal

Reply:
Made this dynamically generated. The test runner will have to have the `ssh-keygen` command available.